### PR TITLE
[ENG-1283] Update schema

### PR DESCRIPTION
**Pull request by adlius**

### CODE REVIEW

File: osf/utils/migrations.py [hunk 1]
```diff
@@ -427,6 +427,25 @@ def describe(self):
...
+
+    def state_forwards(self, app_label, state):
+        pass
+
+    def database_forwards(self, app_label, schema_editor, from_state, to_state):
+        ensure_schemas(to_state.apps)
+        map_schemas_to_schemablocks(to_state.apps)
+
+    def database_backwards(self, app_label, schema_editor, from_state, to_state):
+        warnings.warn('Reversing UpdateRegistrationSchemasAndSchemaBlocks is a noop')
```
Comment (aaxelb) [inline osf/utils/migrations.py]: should this delete all schema blocks?

File: website/project/views/register.py [hunk 2]
```diff
@@ -141,11 +141,6 @@ def node_register_template_page(auth, node, metaschema_id, **kwargs):
                     'message_short': 'Invalid schema name',
                     'message_long': 'No registration schema with that name could be found.'
                 })
-        if not node.registered_schema.filter(id=meta_schema.id).exists():
-            raise HTTPError(http_status.HTTP_400_BAD_REQUEST, data={
-                'message_short': 'Invalid schema',
-                'message_long': 'This registration has no registration supplment with that name.'
-            })
```
Comment (aaxelb) [inline website/project/views/register.py]: i don't understand this case -- trying to look at a specific schema/form for a registration that was not registered with that schema seems like it should be an error, no?

Comment (adlius) [reply website/project/views/register.py]: It seems archaic to me. The reason this fails the test is when there are more than one version of the schema in the db, in which case `meta_schema` will be the newest version and `node.registered_schema` contains the old version.

Comment (adlius) [reply website/project/views/register.py]: Honestly, I don't really care about this now. These are the legacy codes that should be delete anyway. We've put stuff behind feature flags and never ready do away with the flags. If this part of the legacy front end will never be accessed, I'll just delete this and have the tests passed and call it a day, unless there are some other concerns.

**Pull request merged**